OpenRA.Mods.Cnc/Traits/Render/WithLandingCraftAnimation.cs

```diff
@@ -56,7 +56,7 @@ public WithLandingCraftAnimation(ActorInitializer init, WithLandingCraftAnimatio
 
 		public bool ShouldBeOpen()
 		{
-			if (move.CurrentMovementTypes.HasFlag(MovementType.Horizontal) || self.World.Map.DistanceAboveTerrain(self.CenterPosition).Length > 0)
+			if (move.CurrentMovementTypes.HasFlag(MovementType.Horizontal) || move.CurrentMovementTypes.HasFlag(MovementType.Vertical) || move.CurrentMovementTypes.HasFlag(MovementType.Turn) || self.World.Map.DistanceAboveTerrain(self.CenterPosition).Length > 0)
```

Comment:
It would be cleaner to check `move.CurrentMovementTypes != MovementType.None`

Reply:
ok

Reply:
just wait for a second,  I need to look into the code

Reply:
done.

Reply:
I find `HasFlag()` is not useful on `None` here

Reply:
It is functional now